news/2022-09-16-project-loved-september-2022.md

```diff
@@ -0,0 +1,341 @@
+284 BPM, 3:42 | 6.75★, 9.28★\
+*description written by [Horiiizon](https://osu.ppy.sh/users/8071438)*
+
+To finish this round is applerss' map of *Shiawase ni Nareru Kakushi Amenbreak ga Arurashii*, arguably one of the hardest maps ever put up for taiko's Loved voting, coming in at a massive 9.28 stars!
+
+This map is 284 BPM, features heavy and complex 1/6 usage and offers very little breathing room when it comes to rest moments, making it a challenge for even skilled players to just pass the map. Not to worry though, as if you are in the large majority of people who can't pass the Broken difficulty, the Cracked difficulty is significantly easier, containing only 1/4 and much more simplistic patterns.
+
+Do you have what it takes to pass the top diff?
+
+## osu!catch
+
+The osu!catch Loved candidates were chosen by [Sartan](https://osu.ppy.sh/users/4100941), [Tenshichan](https://osu.ppy.sh/users/1101600) and [Wesley](https://osu.ppy.sh/users/2407265)!
+
+---
+
+[![](/wiki/shared/news/2022-09-16-project-loved-september-2022/1234494.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[t+pazolite - Tempestissimo](https://osu.ppy.sh/beatmapsets/1234494#fruits) by [Bunnrei](https://osu.ppy.sh/users/829284)\
+231 BPM, 2:15 | 7.52★\
+*description written by [Wesley](https://osu.ppy.sh/users/2407265)*
+
+This might be a short map, but man is it a banger. But hey, what else can you expect from Bunnrei? The map starts out slowly but quickly turns it up a notch as you can imagine with a song like this. It has a bunch of very quick sliders combined with some stair-esque patterns later on which could throw you off quite easily. To top it all off at the ending, there's just one final slider that will ruin your full combo if you're not paying enough attention!
+
+[![](/wiki/shared/news/2022-09-16-project-loved-september-2022/973421.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[Hatsuki Yura - Kaguyahime](https://osu.ppy.sh/beatmapsets/973421#fruits) by [Kukkai](https://osu.ppy.sh/users/7811952)\
+154 BPM, 4:21 | 6.18★\
+*description written by [Tenshichan](https://osu.ppy.sh/users/1101600)*
+
+*Kaguyahime* is a mapset which has been in graveyard since October 2019, which is a shame because it contains a really clean and well done catch difficulty mapped by Kukkai. Compared to last month's selection of maps this one is way easier to play but still requires a good amount of consistency and speed to be able to get a full combo. It has a lot of fast-paced patterns which may catch you off guard if you aren't ready for them, especially the later half of the map, which could mark the end of a lot of good runs, because that's where it gets increasingly difficult. All in all this is a fun and neat practice map for aspiring players who want to challenge themselves, but also playground for top players to see how they can do with HR.
```

Comment:
```suggestion
*Kaguyahime* is a mapset which has been in graveyard since October 2019, which is a shame because it contains a really clean and well-done catch difficulty mapped by Kukkai.

Compared to last month's selection of maps this one is way easier to play but still requires a good amount of consistency and speed to be able to get a full combo. It has a lot of fast-paced patterns which may catch you off guard if you aren't ready for them, especially the later half of the map, which could mark the end of a lot of good runs, because that's where it gets increasingly difficult.

All in all this is a fun and neat practice map for aspiring players who want to challenge themselves, but also a playground for top players to see how they'll fare with HR.
```